tensorflow_io/core/python/experimental/avro_record_dataset_ops.py

```diff
@@ -16,11 +16,30 @@
 
 import tensorflow as tf
 from tensorflow_io.core.python.ops import core_ops
+from typing import Optional
```

Comment:
Since we are not using any type checker's (like `mypy`) as of now. I feel this style is a bit out of place when compared with other modules in the codebase.

Reply:
@kvignesh1420 Thanks for the comment. Updated, please review.

Reply:
Thanks!